bin/tx-prover/src/commands/update_workers.rs

```diff
@@ -0,0 +1,70 @@
+use clap::Parser;
+use reqwest::Client;
+use serde::{Deserialize, Serialize};
+
+use crate::commands::ProxyConfig;
+
+#[derive(clap::ValueEnum, Clone, Debug, Serialize, Deserialize)]
+pub enum Action {
+    Add,
+    Remove,
+}
+
+impl Action {
+    pub fn as_str(&self) -> &str {
+        match self {
+            Action::Add => "add",
+            Action::Remove => "remove",
+        }
+    }
+}
+
+#[derive(Debug, Parser, Clone, Serialize, Deserialize)]
+pub struct UpdateWorkers {
+    pub action: Action,
+    pub workers: Vec<String>,
+}
+
+impl UpdateWorkers {
+    pub fn execute(&self) -> Result<(), String> {
```

Comment:
Let's add a brief doc comment explaining what this function does (i.e, it makes a request to localhost etc.).

Reply:
I added docs for this method, let me know what do you think.